Cli/Command/AnalyzeCommand.php

```diff
@@ -40,14 +40,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $analysis = $api->analyze($projectUuid, $input->getOption('reference'));
 
         $chars = array('-', '\\', '|', '/');
+        $noAnsiStatus = 'Analysis queued';
+        $output->writeln($noAnsiStatus);
+
         $position = 0;
+
         while (true) {
             // we don't check the status too often
-            if (0 == $position % 2) {
+            if (0 === $position % 2) {
                 $analysis = $api->getAnalysisStatus($projectUuid, $analysis->getNumber());
             }
+
             if ('txt' === $input->getOption('format')) {
-                $output->write(sprintf("%s %-80s\r", $chars[$position % 4], $analysis->getStatusMessage()));
+                if ($input->getOption('no-ansi')) {
```

Comment:
you should use `$output->isDecorated()` instead, to also account for auto-detection of ANSI support